### DISCUSSION TITLE
Make it easy to switch between editor full-size / preview full-size

### DISCUSSION THREAD

**Discussion by danrobi11** [beakerbrowser/beaker#1643] [General]
Hello beaker team
Thank you for new 2.AppImage update. I can now use beaker :)
I got a suggestion here
Would be good to have a quick way to switch the editor to full size.  I tried sliding the editor part to full size and click editor again to see the 'preview mode' but when clicking again on editor it does re-size the editor window to half size. Theres no easy way to deal with this right now.
We could double click the editor top window bar (between 'save' and 'file metadata') to switch the editor between  full size and half size. Or a quick button to make us swtich between editor full size and preview full size.
I know you guys will figure something :)

**Comment by pfrazee** [comment]
> Thank you for new 2.AppImage update. I can now use beaker :)

Good deal

> Would be good to have a quick way to switch the editor to full size. 

That actually does sound like a good idea. Right now one thing you can do is "pop out" the editor into a separate window. [Docs on that are here](https://docs.beakerbrowser.com/beginner/detaching-the-editor).

**Reply by danrobi11** [reply]
Nice! Working fine with "pop out" Thx for the tips!
